libdnf/conf/option_seconds.cpp

```diff
@@ -26,7 +26,7 @@ OptionSeconds::OptionSeconds(ValueType default_value, ValueType min, ValueType m
 
 OptionSeconds::OptionSeconds(ValueType default_value, ValueType min) : OptionNumber(default_value, min) {}
 
-OptionSeconds::OptionSeconds(ValueType default_value) : OptionNumber(default_value) {}
+OptionSeconds::OptionSeconds(ValueType default_value) : OptionNumber(default_value, -1) {}
```

Comment:
What do you think to add to the commit message that the change adds minimal value of the option (-1)?